Cannot convert from 'Spectre.Console.Table' to 'string'

The following code returns the error:

```cs
using Spectre.Console;

namespace NewProject.Services;

public class ContextService : IContextService
{
    private readonly IRootService _rootService;

    public ContextService(IRootService rootService)
    {
        _rootService = rootService ?? throw new ArgumentNullException(nameof(rootService));
    }

    public void List()
    {
        var root = _rootService.GetPath();
        DirectoryInfo info = new DirectoryInfo(root);
        var directories = info.EnumerateDirectories();

        var table = new Table();
        table.Border = TableBorder.SimpleHeavy;
        table.AddColumn("Context");
        table.AddColumn("Path");

        foreach (var item in directories)
        {
            table.AddRow(item.Name, item.FullName);
        }

        AnsiConsole.Write(table); // Here is C:\Repos\NewProject\Services\ContextService.cs(36,27)
    }
}
```

The build output:

```
C:\Repos\NewProject\Services\ContextService.cs(36,27): error CS1503: Argument 1: cannot convert from 'Spectre.Console.Table' to 'string' [C:\Repos\NewProject\NewProject.csproj]

The build failed. Fix the build errors and run again.
```

Could anyone help me?

All right, so the Program.cs looks like this:

```cs
using Microsoft.Extensions.DependencyInjection;
using Spectre.Console;
using Spectre.Console.Cli;

public static class Program
{
    public static int Main(string[] args)
    {
        var app = new CommandApp();
        app.Configure(config =>
        {
            config.AddCommand<ListCommand>("list");
            config.PropagateExceptions();
        });

        try
        {
            return app.Run(args);
        }
        catch (Exception ex)
        {
            AnsiConsole.WriteException(ex, ExceptionFormats.ShortenEverything);
            return -99;
        }
    }
}
```

The .csproj file looks like this:

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net6.0</TargetFramework>
    <RootNamespace>test_spectre</RootNamespace>
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.0" />
    <PackageReference Include="Spectre.Console" Version="0.45.0" />
  </ItemGroup>

</Project>
```

The output of the build process contains: `C:\Users\gjary\Repos\test-spectre\Program.cs(3,23): error CS0234: The type or namespace name 'Cli' does not exist in the namespace 'Spectre.Console' (are you missing an assembly reference? [C:\Users\gjary\Repos\test-spectre\test-spectre.csproj]`.

--- a/docs/input/quick-start.md
+++ b/docs/input/quick-start.md
@@ -8,9 +8,10 @@ to install the NuGet package.
 
 ```text
 > dotnet add package Spectre.Console
+> dotnet add package Spectre.Console.Cli
 ```
 
-After that you will need to reference the `Spectre.Console` namespace.
+After that you will need to reference the `Spectre.Console` and `Spectre.Console.Cli` namespaces.
 Once that is done, you can start using all the available features.
 
 ```csharp

